test/github.ts

```diff
@@ -703,13 +703,6 @@ describe('GitHub', () => {
   });
 
   describe('createRelease', () => {
-    let githubCreateReleaseSpy: sinon.SinonSpy;
-    beforeEach(async () => {
-      githubCreateReleaseSpy = sandbox.spy(
-        github['octokit'].repos,
-        'createRelease'
-      );
-    });
```

Comment:
We can no longer spy on this method call using sinon. These checks were already unnecessary as we snapshot the body of the http request in each of the 3 tests that were using this spy.